tests/test_torch_musa_unsupport.py

```diff
@@ -0,0 +1,1685 @@
+import torch
+import torch_musa
```

Comment:
what is this file used for? and what's the difference between torch and torch_musa? I thought most of the operators listed here are supported.

Reply:
torch_musa implemented the musa backend and not up-streamed to the trunk of torch, behaves as a plugin here.